### ISSUE TITLE
Caching of EXIF

### ISSUE THREAD

**Issue by cfjedimaster** [exif-js/exif-js#52]
Given an image in the dom where we switch the src dynamically, it looks like EXIF will cache the tags and not reload it.

**Comment by NJseo** [comment]
Use a cache buster. Append a time variable to the end of the source.

image.jpg?tm=time();

**Comment by cfjedimaster** [comment]
Yeah, that would help. Still seems like the code itself should have a way to handle this though.

**Comment by ravi-nextbit** [comment]
There should still be a way from the library to force a reload, because using a cache buster will disable all caching. So +1 for this feature. 
I solved this issue by removing the img tag and adding a new one, this allows you to use browser caching while still forcing this library to reload.

**Comment by liangmicha** [comment]
I solved this by setting `exifdata` to `null` before the `EXIF.getData`
```
      var imgTag = $('.thumbnail > img')[0];
      imgTag.exifdata = null;

      EXIF.getData(imgTag, function() {
        var allMetaData = EXIF.getAllTags(this);
        console.log(allMetaData);
      });
```
